src/pretix/control/templates/pretixcontrol/item/index.html

```diff
@@ -186,7 +186,7 @@
                             {% bootstrap_field form.media_type layout="control" %}
                         {% endif %}
                         {% for f in plugin_forms %}
-                            {% if f.is_layouts %}
+                            {% if f.is_layouts and not f.title %}
```

Comment:
Hmm, a form with ``f.is_layouts and f.title`` would now not be rendered anywhere, do I see that correctly?

Reply:
That is correct: there are some invalid combinations.

However, rather than fail silently and not render the form, an exception will be thrown for such forms, with a helpful error message:

https://github.com/pretix/pretix/blob/bf40956a2d1f8cbb0efe43eea250e2b7b5bc1313/src/pretix/control/views/item.py#L1302-L1306

Reply:
> However, I believe "template but no title" could and should work with only marginal effort?

I suppose if no `title` is provided, then the form should not have it's own tab, but

```
{% bootstrap_form f layout="control" %}
```

could be replaced with

```
{% include f.template with form=f %}
```

Is this what you have in mind?

Reply:
Yes, exactly!

Reply:
Done!